Validation/Configuration/python/valid3Dt.py

```diff
@@ -0,0 +1,17 @@
+import FWCore.ParameterSet.Config as cms
+
+def customise(process):
```

Comment:
where will this be used?
shouldn't the changes become a default (or era-dependent) for `mtdTracksValid` and `vertices4DValid` ?

Reply:
this is not meant to become default, but to be used for studies in MTD on the performance of the 3Dt algorithm, that can be compared to the new default version, the 3Dt + 4D algorithm